### PR TITLE
This case was never ported to qtFRED

### PR DESCRIPTION
**Pull request by MjnMixael**

### CODE REVIEW

**Review by JohnAFernandez — APPROVED** [review]

File: qtfred/src/ui/widgets/sexp_tree.cpp [hunk 1]
```diff
@@ -1346,6 +1346,7 @@ int sexp_tree::get_default_value(sexp_list_item* item, char* text_buf, int op, i
 	case OPF_SUBSYSTEM:
 	case OPF_AWACS_SUBSYSTEM:
 	case OPF_ROTATING_SUBSYSTEM:
+	case OPF_TRANSLATING_SUBSYTEM:
```
Comment (tomimaki) [inline qtfred/src/ui/widgets/sexp_tree.cpp:1349]: Subsytem is typo or intended?

Comment (MjnMixael) [reply qtfred/src/ui/widgets/sexp_tree.cpp]: Typo. Good catch.

**Review by Goober5000 — CHANGES_REQUESTED** [review]
Great catch.

Looking at `get_listing_opf_subsystem`, there are a few additional edits, related to `OPS_TRANSLATE`, that are needed in order to fully sync the two files.

**Review by Goober5000 — CHANGES_REQUESTED** [review]
Almost done.  The `OPS_TRANSLATE` definition needs to be added to the `// specific types of subsystems we're looking for` list, in the same order as in FRED's sexp_tree.cpp.

**Comment by MjnMixael** [comment]
> Almost done. The `OPS_TRANSLATE` definition needs to be added to the `// specific types of subsystems we're looking for` list, in the same order as in FRED's sexp_tree.cpp.

yeah ok. I'm kinda done cleaning up the qtFRED mess. Feel free to edit this PR as you like. I have neither the time or motivation to keep chasing this one in circles and I was told recently that's all the reason I need not to do stuff.

**Comment by MjnMixael** [comment]
OK fine. Last one because I found another thing missing.

**Review by Goober5000 — APPROVED** [review]

**Pull request merged**